contracts/distribution.js

```diff
@@ -203,10 +198,25 @@ actions.update = async (payload) => {
   if (api.assert(authorizedCreation, 'you must have enough tokens to cover the update fee')
     && api.assert(isSignedWithActiveKey === true, 'you must use a transaction signed with your active key')) {
     const exDist = await api.db.findOne('batches', { _id: id });
-    if (api.assert(exDist, 'distribution not found') && await validateMinPayout(tokenMinPayout)
-      && await validateRecipients(tokenRecipients)) {
-      exDist.tokenMinPayout = tokenMinPayout;
-      exDist.tokenRecipients = tokenRecipients;
+    if (api.assert(exDist, 'distribution not found')) {
+      if (numTicks && api.assert(typeof numTicks === 'string' && api.BigNumber(numTicks).gt(0) && api.BigNumber(numTicks).lte(5555), 'numTicks must be a number between 1 and 5555')) {
+        exDist.numTicks = numTicks;
```

Comment:
If numTicks is updated, should numTicksLeft be updated as well?

Reply:
Yes it should. Updating numTicks will always restart the payment schedule.